packages/relay/src/lib/relay.ts

```diff
@@ -35,11 +35,9 @@ import { Web3Impl } from './web3';
 
 export class Relay {
   /**
-   * @private
-   * @readonly
-   * @property {Client} clientMain - The primary Hedera client used for interacting with the Hedera network.
+   * The primary Hedera client used for interacting with the Hedera network.
    */
-  private readonly clientMain: Client;
+  private readonly operatorAccountId: AccountId | null;
```

Comment:
q: can we transform the `AccountId` SDK type to a string, use only the string representation, and get rid of the SDK-specific types? Or are we using methods from `AccountId` object? 

It can be applicable for the `PublicKey` [here](https://github.com/hiero-ledger/hiero-json-rpc-relay/pull/4210/files#diff-4b0bcc39d9efb3e923f041c7459f1cc42d34597d4200839df6764c2a345a1c1bR123).

Reply:
That's a good point. I'll open a new issue as a follow up for this.